app/routes/__frontend/resources/virtual-coffee-handbook/guides-to-virtual-coffee/lunch-and-learns.mdx

```diff
@@ -0,0 +1,55 @@
+---
+meta:
+  title: Lunch & Learns
+  description: Share your thoughts and ideas with a small audience of friends!
+hero:
+  Hero: UndrawPresentation
+tags: [memberresources]
+order: 6
+---
+
+import LeadText from '~/components/content/LeadText';
+import TextContainer from '~/components/content/TextContainer';
+import { Link } from '@remix-run/react';
+
+<TextContainer background="light" showBackToTopLink={false}>
+
+<LeadText>
+
+Lunch & Learn talks are usually hour-long sessions on one topic. It can be a traditional conference-style talk, panel discussion, question and answer, a combination, or anything else you can think of!
+
+</LeadText>
+
+</TextContainer>
+
+## Table of Contents
+
+<TextContainer>
+
+---
+
+## What Is a Lunch & Learn?
+
+Lunch & Learn is a speaking opportunity for anyone in the community. If you want to talk about something the Virtual Coffee community finds interesting, we have members willing to listen and ask questions. You don't have to have given a talk before, you don't have to be an expert — if there is such a thing — and you don't need to worry about whether or not your talk will be accepted. We're here to support everyone where they are in the speaking experience.
```

Comment:
A Lunch & Learn is a speaking opportunity for anyone in the community. If you want to talk about something, we have members willing to listen and ask questions. No public speaking experience required. You don't have to be an expert (if there is such a thing) and you don't need to worry about whether or not your talk will be accepted. We're here to support everyone where they are in their public speaking journey.